main.py

```diff
@@ -22,11 +22,23 @@
 import flask_login as login
 from flask_admin.contrib import sqla
 from flask_admin import helpers, expose, BaseView
-from werkzeug.security import check_password_hash
+from werkzeug.security import generate_password_hash, check_password_hash
+from flask_mail import Mail,Message
 
 app = utilities.init_flask(__name__)
+app.config.from_object(__name__)
+app.config.update(dict(
+    MAIL_SERVER='smtp.gmail.com',
+    MAIL_PORT=465,
+    MAIL_USE_TLS = False,
+    MAIL_USE_SSL= True,
+    MAIL_USERNAME = 'anywaytesttest@gmail.com',
```

Comment:
Yes, it's in Heroku. You can use environment variables for the credentials,
and then we'll set them in the production server.

On Tue, May 19, 2015 at 11:32 AM Avi Klaiman <notifications@github.com>
wrote:

> In main.py
> <https://github.com/hasadna/anyway/pull/258#discussion_r30578747>:
>
> >
> >  app = utilities.init_flask(__name__)
> > +app.config.from_object(__name__)
> > +app.config.update(dict(
> > +    MAIL_SERVER='smtp.gmail.com',
> > +    MAIL_PORT=465,
> > +    MAIL_USE_TLS = False,
> > +    MAIL_USE_SSL= True,
> > +    MAIL_USERNAME = 'anywaytesttest@gmail.com',
>
> I set up this email just to check if it woks and it does.
> But I can't write the details of the real email here because then it can
> be hacked.
> Who has access to the release version? it's in heroku, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hasadna/anyway/pull/258/files#r30578747>.
>